docs/sql-ref-syntax-qry-select-groupby.md

```diff
@@ -81,6 +81,8 @@ aggregate_name ( [ DISTINCT ] expression [ , ... ] ) [ FILTER ( WHERE boolean_ex
 * **aggregate_name**
 
     Specifies an aggregate function name (MIN, MAX, COUNT, SUM, AVG, etc.).
+    Some aggregate function like `FIRST` and `LAST` have special usage as the following:
+    **Syntax:** `[ FIRST | LAST ] ( [ distinct ] expression [ IGNORE NULLS ] ) [ FILTER ( WHERE boolean_expression ) ]`
```

Comment:
```
* **aggregate_name**

Specifies an aggregate function name (MIN, MAX, COUNT, SUM, AVG, FIRST, LAST, etc.).
Note that `FIRST` and `LAST` have an optional `IGNORE NULLS` clause; when the option specified, it will returns the first or last value that is not null (or null if all values are null). 

* **DISTINCT**
```